### ISSUE TITLE
How to delete tasks?

### ISSUE THREAD

**Issue by joto** [maproulette/maproulette2#245]
Can I delete all taks from a challenge?

**Comment by mvexel** [comment]
Currently the only way would be to delete and re-create the challenge, or to delete them one by one. Is What is your use case?

**Comment by joto** [comment]
Use case: Starting over after I discover that tasks are bad. Or maybe they are all done or so. Re-creating the challenge means I have to add metadata for the challenge again and that the new challenge has a new ID. If I have published the link to the challenge, I can't use it any more.

**Comment by mvexel** [comment]
Fair enough. Would that make sense as `DELETE /api/v2/challenge/:id/tasks` ?
